### DISCUSSION TITLE
Missing gvmd-log Logfile and GVM_DATA

### DISCUSSION THREAD

**Discussion by devMarc5598** [Secure-Compliance-Solutions-LLC/GVM-Docker#286] [Q&A]
Dear all,

My Greenbone is running but i have no gvmd-log file in /var/log/gvm/ and the one in /var/log/supervisor/ is empty...
And as already mentioned in  Missing GVM_DATA in GSA #273  (https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/discussions/273) the GVM_DATA is also missing... 
I got this data in /var/lib/gvm/data-objects/gvmd but not in the GSA. 
I think that  greenbone-feed-sync --type GVMD_DATA does nothing on my deployment because I have to get the data with  rsync -avrP rsync://feed.community.greenbone.net:/data-objects/gvmd/ .

Any ideas?
I think gvmd doesnt load the data in the postgresdb. So GSA has no data to show.

**GVM Versions:**
gsad: 21.4.2
gvmd: 21.4.3
openvas-scanner: 21.4.2
gvm-libs: 21.4.2

**Comment by austinsonger** [comment]
Can you create an issue please. This shouldn't be a discussion.

**Reply by devMarc5598** [reply]
Okay. I opened an issue:  "Missing gvmd-log Logfile and GVM_DATA #287"

**Comment by Dexus** [comment]
Try to mod the feed.xml and update the timestamp + 1

Von meinem iPhone gesendet
